DataConnectors/BETTER MTD.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
No space in file name please. 
Rename the connector file name to BETTERMTD.json

Reply:
Renamed